packages/block-editor/src/components/global-styles/get-block-css-selector.js

```diff
@@ -31,7 +31,7 @@ export function getBlockCSSSelector(
 	const { fallback = false } = options;
 	const { name, selectors, supports } = blockType;
 
-	const hasSelectors = ! isEmpty( selectors );
+	const hasSelectors = selectors && Object.keys( selectors ).length > 0;
```

Comment:
`selectors` are optional, but if declared, are expected to be an object:

https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#selectors